docs/contributing/coding-guidelines/ros-nodes/.pages

```diff
@@ -5,6 +5,7 @@ nav:
   - coordinate-system.md
   - directory-structure.md
   - launch-files.md
+  - Message guidelines: message-guidelines.md
```

Comment:
The title here can be omitted.

```suggestion
  - message-guidelines.md
```

Reply:
Because the page title is used.
![image](https://user-images.githubusercontent.com/31987104/167392427-83900567-7c6e-46fc-90ee-b4b614a829a3.png)

Reply:
Ok, done in https://github.com/autowarefoundation/autoware-documentation/pull/94/commits/56e8eab84a2a7584825cfa45769e0de4ee4ac343